### PR TITLE
Publish 02/03/2023 3:30 PM PT

### PR DESCRIPTION
**Pull request by AngelaMotherofDragons**

### CODE REVIEW

**Comment by AngelaMotherofDragons** [comment]
@chrisda -
cc: @traya1 

This publish PR has a warning. Would you confirm that the file needs deleting and not the redirect entry? Once you confirm, if you let me know then I can make the change or you can resolve the warning and let me know that I can merge to live.

Thanks!

**Comment by chrisda** [comment]
@AngelaMotherofDragons, that must be a red herring error; there iwas no "Set-PhishFilterPolicy.yml"; the file was Set-PhishFilterPolicy.md", and that's what's in the redirect entry.

**Comment by AngelaMotherofDragons** [comment]
@traya1 - What do you advise to resolve this issue of the ghost file?

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -6839,6 +6839,21 @@
...
       "redirect_document_id": false
+    },
+    {
+      "source_path": "exchange/virtual-folder/exchange/Get-MailTrafficTopReport.md",
+      "redirect_url": "/powershell/module/exchange/get-mailtrafficsummaryreport",
+      "redirect_document_id": false
+    },
+    {
+      "source_path": "exchange/virtual-folder/exchange/Get-PhishFilterPolicy.md",
+      "redirect_url": "/powershell/module/exchange/get-tenantallowblocklistppoofitems",
```
Comment (JohanFreelancer9) [inline .openpublishing.redirection.json:6850]:
```suggestion
      "redirect_url": "/powershell/module/exchange/get-tenantallowblocklistspoofitems",
```

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -6839,6 +6839,21 @@
...
+      "redirect_document_id": false
+    },
+    {
+      "source_path": "exchange/virtual-folder/exchange/Set-PhishFilterPolicy.md",
+      "redirect_url": "/powershell/module/exchange/set-tenantallowblocklistppoofitems",
```
Comment (JohanFreelancer9) [inline .openpublishing.redirection.json:6855]:
```suggestion
      "redirect_url": "/powershell/module/exchange/set-tenantallowblocklistspoofitems",
```

File: exchange/exchange-ps/exchange/Get-UMCallSummaryReport.md [hunk 3]
```diff
@@ -39,12 +39,12 @@ The Get-UMCallSummaryReport cmdlet displays the aggregated statistics about all
...
-- Failed Or Rejected: Percentage of calls that either failed or were rejected by the Mailbox server for that organization.
+- Failed Or Rejected: Percentage of calls that either failed or were rejected by the server for that organization.
 - Audio Quality: Overall audio quality for the selected period of time for the organization/user. 4.50 or higher = Excellent, 3.5 to 4.49 = Good, 2.5 to 3.49 = Average, 1.50 to 2.49 =Poor, and 1.49 or lower = Bad.
 - Total Calls: If the UM IP gateway is selected, this is the total number of calls grouped for the selected UM IP gateway for the corresponding date, If the UM dial plan control is selected, this is the total number of calls grouped for the selected UM dial plan for the corresponding date, and If the user is selected, this column has the total number of calls for the user.
 - Network MOS (NMOS): Average NMOS for the specific UM dial plan or UM IP gateway.
-- NMOS Degradation:
-- NMOS degradation for the specific UM dial plan or UM IP gateway.:
+- NMOS Degradation.
+- NMOS degradation for the specific UM dial plan or UM IP gateway.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Get-UMCallSummaryReport.md:47]:
```suggestion
- NMOS Degradation: NMOS degradation for the specific UM dial plan or UM IP gateway.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AngelaMotherofDragons, kindly commit the suggested changes. Many thanks.

**Comment by traya1** [comment]
> * Set-PhishFilterPolicy.yml

The above topic does not exist in the repo. Will merge the PR and monitor the builds.

**Pull request merged**